src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -203,749 +203,114 @@ private ImmutableArray<ISwitchCase> GetPatternSwitchStatementCases(BoundPatternS
             });
         }
 
-        private static BinaryOperationKind GetLabelEqualityKind(BoundSwitchLabel label)
+        internal class Helper
```

Comment:
>Helper [](start = 23, length = 6)

Perhaps we should give this class a more meaningful name and move it to its own file, or should get rid of it and place methods into the containing type directly.